packages/config/config/devices/0x013c/pst02a.json

```diff
@@ -61,47 +60,309 @@
 				}
 			]
 		},
-		"5": {
-			"label": "Operation Mode",
-			"description": "Mode of operation and enabled multisensor functions",
+		"5[0x02]": {
+			"label": "Test Mode",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
+			"maxValue": 1,
 			"defaultValue": 0,
+			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Set Celsius",
-					"value": 8
+					"label": "Disable",
+					"value": 0
 				},
 				{
-					"label": "Preset: Celsius and LED on = Bits: 00001010 = 10",
-					"value": 10
+					"label": "Enable",
+					"value": 1
 				}
 			]
 		},
-		"6": {
-			"label": "Multi-Sensor Function Switch",
-			"description": "Enable or disable functions of the multisensor.",
+		"5[0x04]": {
+			"label": "Door/Window Function",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 4
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 1
+				},
+				{
+					"label": "Enable",
+					"value": 0
+				}
+			]
 		},
-		"7": {
-			"label": "Customer Function",
-			"description": "Enable or disable functions of the multisensor.",
+		"5[0x08]": {
+			"label": "Temperature Scale",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 20,
```

Comment:
The old parameter 7 had a default value of 20, which means that the default for 7[0x10] and 7[0x04] should be 1 if I'm not mistaken.

Reply:
According to this document, the default value is 4: https://doc.eedomus.com/files/PST02_Manual-2014-09-17.pdf